backend/apis/clients/client-logs/get-activity-logs.api.js

```diff
@@ -23,7 +23,7 @@ app.get("/api/clients/:clientId/logs", (req, res, next) => {
       users.id createdById, users.firstName createdByFirstName, users.lastName createdByLastName
     FROM clientLogs JOIN users
     ON clientLogs.addedBy = users.id
-    WHERE clientLogs.clientId = ?
+    WHERE clientLogs.clientId = ? AND isDeleted = false
```

Comment:
Should this be "true"?

Reply:
No. The `GET /api/clients/:id/logs` endpoint should only return logs that are **not deleted**